.env.example

```
@@ -0,0 +1,22 @@
+# The Flying Bus - Environment Configuration Template
+# Copy this file to .env and fill in your actual values
+
+# Thirdweb Configuration
+# Get these from https://thirdweb.com/dashboard
+VITE_THIRDWEB_CLIENT_ID=your_thirdweb_client_id_here
+VITE_THIRDWEB_SECRET_KEY=your_thirdweb_secret_key_here
+VITE_WALLET_PRIVATE_KEY=your_wallet_private_key_here
+VITE_POLYGON_RPC_URL=https://polygon-rpc.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_POLYGON_RPC_URL key should go before the VITE_THIRDWEB_CLIENT_ID key

(UnorderedKey)